libs/liquidity/src/lib/liquidity-table.tsx

```diff
@@ -154,15 +161,75 @@ export const LiquidityTable = ({
         tooltipValueGetter: stakeToCcyVolumeFormatter,
       },
       {
-        headerName: t('Status'),
-        headerTooltip: t('The current status of this liquidity provision.'),
-        field: 'status',
-        valueFormatter: ({ value }) => {
-          if (!value) return value;
-          return LiquidityProvisionStatusMapping[
-            value as LiquidityProvisionStatus
-          ];
-        },
+        headerName: t(`(Current) Fraction on the book`),
+        field: 'sla.currentEpochFractionOfTimeOnBook',
+        type: 'rightAligned',
+        headerTooltip: t('Current epoch fraction of time on the book.'),
+        valueFormatter: percentageFormatter,
+      },
+      {
+        headerName: t(`(Last) Fraction on the book`),
+        field: 'sla.lastEpochFractionOfTimeOnBook',
+        type: 'rightAligned',
+        headerTooltip: t('Last epoch fraction of time on the book.'),
+        valueFormatter: percentageFormatter,
+      },
+      {
+        headerName: t(`(Last) Fee penalty`),
+        field: 'sla.lastEpochFeePenalty',
+        type: 'rightAligned',
+        headerTooltip: t('Last epoch fee penalty.'),
+        valueFormatter: percentageFormatter,
+      },
+      {
+        headerName: t(`(Last) Bond penalty`),
+        field: 'sla.lastEpochBondPenalty',
+        type: 'rightAligned',
+        headerTooltip: t('Last epoch bond penalty.'),
+        valueFormatter: percentageFormatter,
+      },
+      // {
```

Comment:
I get an array of values for the `Hysterisis period fee penalties` and no values for the rest of the commented columns. 

We should find a different way to display the first commented column.